lib/blocks.php

```diff
@@ -178,6 +178,10 @@ function gutenberg_register_core_block_styles( $block_name ) {
 			filemtime( gutenberg_dir_path() . $style_path )
 		);
 		wp_style_add_data( "wp-block-{$block_name}", 'rtl', 'replace' );
+
+		// Add a reference to the stylesheet's path to allow calculations for inlining styles in `wp_head`.
+		global $wp_styles;
+		$wp_styles->registered[ "wp-block-{$block_name}" ]->path = gutenberg_dir_path() . $style_path;
```

Comment:
Could you use `wp_style_add_data` to set path instead? Just asking, since it is very similar to `rtl`.

Reply:
I hadn't thought of that... Done in https://github.com/WordPress/gutenberg/pull/28358/commits/1219cae530131c2c00e907e753810cfc784be130 :+1: